src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/ProcessTests.cs

```diff
@@ -187,36 +187,31 @@ public void TestMachineName()
             Assert.NotNull(_process.MachineName);
         }
 
-        [Fact]
+        [Fact, PlatformSpecific(~PlatformID.OSX)]
```

Comment:
If we're going to make this test not for OS X, should we add a separate test that's specific to OS X and that verifies nothing bad happens when accessing MainModule? e.g. ```Assert.Null(_process.MainModule);```